.github/workflows/track_backlog.yml

```diff
@@ -0,0 +1,26 @@
+name: Track new issues in backlog
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "45 0 * *" # Run at 45 minutes past midnight
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Track all issues
+      uses: dhruvkb/issue-projector@0.0.5
+      with:
+        ACCESS_TOKEN: ${{ secrets.ADMIN_GITHUB_TOKEN }}
+        ORG_NAME: 'creativecommons'
+        PROJECT_NUMBER: 10 # Backlog
+        COLUMN_NAME: 'Pending Review'
+        EXCLUDED_PROJECT_NUMBER: 7 # Active sprint
+        ISSUE_TYPE: 'issue'
+    - name: Track all PRs
+      uses: dhruvkb/issue-projector@0.0.5
+      with:
+        ACCESS_TOKEN: ${{ secrets.ADMIN_GITHUB_TOKEN }}
+        ORG_NAME: 'creativecommons'
+        PROJECT_NUMBER: 7 # Active sprint
+        COLUMN_NAME: 'In Progress (Community)'
```

Comment:
Can we put these in Code Review instead?

Reply:
Changed column to 'Code Review'.